### ISSUE TITLE
What is the resource ID for a cloud storage bucket?

### ISSUE THREAD

**Issue by pbiggar** [GoogleCloudPlatform/k8s-config-connector#485]
I want to export the config-connect yaml for an existing Cloud Storage Bucket. The docs describe running `config-connector export [URL]`, however it's not clear what the URL should be for a bucket.

The docs link to https://cloud.google.com/iam/docs/full-resource-names, which suggests `//storage.googleapis.com/projects/_/buckets/bucket-id`. I tried this with:

`config-connector export //storage.googleapis.com/projects/_/buckets/config-connector-experiment`

and got

```
error in 'config-connector' version '1.51.2': error getting unstructured: error getting next unstruct: error converting url '//storage.googleapis.com/projects/_/buckets/config-connector-experiment' to skeleton: error getting service mapping and resource config for url '//storage.googleapis.com/projects/_/buckets/config-connector-experiment': unable to find a matching resource config for uri path '/projects/_/buckets/config-connector-experiment'
```

The docs have a footnote that says "1 The format shown here is compatible with IAM. Some services use a different format for this resource type." PErhaps there's a different format for Config Connector? If so, what is it, and can it be added to docs?

thanks!

**Comment by toumorokoshi** [comment]
Hi! Thanks for the question.

I took a look through the code and it looks like the storage resources are not supported in config-connector CLI. Sorry about the confusing error message.

I'm taking a look to see if there's something we can do, minimum improving the error message, and at best adding support.

**Comment by pbiggar** [comment]
Thanks for letting me know! Just to be clear, you're saying you can't export Cloud Storage resources directly from the CLI, but I can still hand-write the config-connector yaml?

**Comment by toumorokoshi** [comment]
> Thanks for letting me know! Just to be clear, you're saying you can't export Cloud Storage resources directly from the CLI, but I can still hand-write the config-connector yaml?

Correct! You can always look at the [resource supported in the reference](https://cloud.google.com/config-connector/docs/reference/overview). For convenience inlining the current list:

- StorageBucket
- StorageBucketAccessControl
- StorageDefaultObjectAccessControl
- StorageNotification

**Comment by toumorokoshi** [comment]
Also as an update: i think it's a non-trivial change to support these resources, so we should mark a feature request. I'll convert this question into a feature request if that sounds ok with you?

**Comment by pbiggar** [comment]
Sure, thank you!

**Comment by ldanielmadariaga** [comment]
This item has been open for >12 months. As part of scheduled repository maintenance, this Issue/PR will be closed. If this is still a critical ask, please re-open and work with your Google Cloud Account team to prioritize it with GCP KCC Product Management.
